CRM/Civicase/Service/CaseCategoryCustomFieldsSetting.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Case Category Custom Fields Setting class.
+ */
+class CRM_Civicase_Service_CaseCategoryCustomFieldsSetting {
```

Comment:
Any reason why we are using static functions throughout this class? I think we should make them all member functions. This is a proper service and will be better to have the methods as member functions. We will be able to add a constructor later and these methods will have access to whatever variable/property is set in the constructor.


Also let's add Unit tests for this class. You can take a look at the tests for the finance management setting tests class for a cue.

Reply:
Since there were no values shared between them I thought static functions were enough. Could go for passing the category ID to the constructor, but will skip for now.